android/app/src/main/java/app/organicmaps/MwmActivity.java

```diff
@@ -2276,14 +2263,10 @@ private void startTrackRecording(boolean toast)
       return;
     }
 
-    // Check for battery saver permission
-    if (!requestBatterySaverPermission(PowerSaveDisclaimerState.SHOWING_FOR_RECORDING))
-      return;
-
+    requestBatterySaverPermission();
     requestPostNotificationsPermission();
```

Comment:
Here is one issue. Since both are non blocking notifications both are called simultaneously due to which the battery saver permission do not stays and gets removed by post notification permission request. I have tested it in android 14 device.

Steps to regenerate
1- Use android 13 or above phone
2- Remove notification permission for OM and also turn on battery saver
3- Start the trace path feature in OM.
4- for a second dialog to disable power saver will appear and then it will be removed and post notification permission dialog will appear.

Another not much important concern is about asking post notification permission again and again on every start of track recorder. Since the UX logic for battery saver is to ask only once then it would be better to design this in same manner as well. Although android do not let the app ask post notification permission again and again if once it has been denied by user but i am proposing it for OM's UX : )